### ISSUE TITLE
Unable to run it

### ISSUE THREAD

**Issue by FlavioSuarez** [neznaika0/codeigniter-expenses#1]
Hi,

Giving a try to your app, but I'm unable to run it...
Tried with several CI versions, on development mode, but I get "CodeIgniter\Security\Exceptions\SecurityException #403"
I tried on Laragon and also with spark serve with the same result...

Any advice?
TIA!

**Comment by neznaika0** [comment]
Thank you for your interest in the project. 

How do you run the server?
What was changed after copying?

The 403 error may appear due to **csrf** - when you submit the form again or after a while

**Comment by FlavioSuarez** [comment]
Hi,

Thank you for your response,
Got it work!
The problem was the "cookie.secure = true" on the env file. Setting it to "false" solved the problem.

Instead of XAMP, I use Laragon as development environment.  On Laragon I don't got it work, but YES using the "spark serve".

But then, when I tried to run Statistic from the menu I got another error: 
CodeIgniter\Database\Exceptions\DatabaseException Expression #2 of SELECT list is not in GROUP BY clause and contains nonaggregated column which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by

This was solved by changing the sql mode in MySQL by this command:
SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));

So, it's all running now and I can learn with your app! 
Thank you for sharing!
Best regards
